ocs/defaults.py

```diff
@@ -0,0 +1,4 @@
+"""
+Defaults module
+"""
+INSTALLER_VERSION = '4.1.0-rc.0'
```

Comment:
@RazTamir I am just thinking about use yaml format for the defaults, Here we can load yaml format to DEFAULTS variable and from other modules we can do.

```python
from defaults import DEFAULTS

ver = config.get('installer-version', DEFAULTS['INSTALLER_VERSION'])
```

Advantage for it is that we can easily use it when rendering templates.
So we can do some other helper function which can do something like this:

```python
def get_updated_defaults(data={}):
"""
Returns updated defaults with provided data

Args:
    data (dict): Dictionary with new data (default: {})

Returns:
    dict: Default values with updated data
"""
defaults_copy = DEFAULTS.copy()  # deep copy maybe better here 
return defaults_copy.update(data)
```

so once rendering some template we can take all the data from defaults easily instead of naming them explicitly and just update with new data we need? WDYT about this approach?

Reply:
@petr-balogh sounds good to me.
I'll abandon this change.

I won't be able to implement this new approach soon so if someone can take this task it will be great

Reply:
Maybe no need to abandon, we can start with what you have and just in the comment you can point to this discussion as TODO, so this can be done in new PR.

Reply:
Ok, created this for future conversion to yaml

Reply:
@RazTamir ok, thanks, but let's also other ppl to give their 2 cents here. Maybe it's not good idea for all. @clacroix12 @vasukulkarni @dahorak @vavuthu @shylesh @ebenahar .

Reply:
Taking this up in a next PR